packages/react/src/primitives/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+
+import {
+  BaseBreadcrumbProps,
+  BreadcrumbsProps,
+  Primitive,
+  ForwardRefPrimitive,
+} from '../types';
+
+import { BreadcrumbItem } from './BreadcrumbItem';
+import { BreadcrumbLink } from './BreadcrumbLink';
+import { BreadcrumbSeparator } from './BreadcrumbSeparator';
+import { BreadcrumbContainer } from './BreadcrumbContainer';
+
+const DefaultSeparator = <BreadcrumbSeparator>{'/'}</BreadcrumbSeparator>;
```

Comment:
What do you think of using `"/"` as the default value of `children` in `BreadcrumbsSeparator` so in composable mode the default `Breadcrumbs.Separator` renders the slash?